src/lipalightninglib.udl

```diff
@@ -76,25 +76,25 @@ interface LightningNode {
     //          * have not expired
     //    - metadata - a metadata string that gets tied up to this payment. It can be used by the user of this library
     //  to store data that is relevant to this payment. It is provided together with the respective payment in `get_latest_payments()`
-    [Throws=LnError]
+    [Throws=PayError]
     void pay_invoice(string invoice, string metadata);
 
     // Differs from pay_invoice(string invoice, string metadata) in that the passed in invoice does not have any payment amount specified,
     // and allows the caller of the method to specify an amount instead.
     //
     // Additional Parameters:
     //    - amount_sat - amount in sats to be paid
-    [Throws=LnError]
+    [Throws=PayError]
     void pay_open_invoice(string invoice, u64 amount_sat, string metadata);
 
     // Get a list of the latest payments
     //    - number_of_payments - the maximum number of payments that will be returned
-    [Throws=LnError]
+    [Throws=PayError]
```

Comment:
Maybe this is just due to the WIP nature of the PR, but why would `PayError` be the type here? The same question would apply to `get_payment()` and `create_invoice()`.

Reply:
Yes that was a bit prototype and I didn't know how I wanted to split the errros at first. Should be fixed now.